api/src/legacy_soap_api/legacy_soap_api_client.py

```diff
@@ -114,7 +114,27 @@ def get_simpler_soap_response(self, proxy_response: SOAPResponse) -> tuple:
         """
         proxy_response_soap_dict = self.get_proxy_soap_response_dict(proxy_response)
         simpler_response_soap_dict = self.get_soap_response_dict()
-        diff_soap_dicts(simpler_response_soap_dict, proxy_response_soap_dict)
+
+        try:
+            diff_results = diff_soap_dicts(
+                sgg_dict=get_envelope_dict(
+                    simpler_response_soap_dict, self.operation_config.response_operation_name
+                ),
+                gg_dict=get_envelope_dict(
+                    proxy_response_soap_dict, self.operation_config.response_operation_name
+                ),
+                key_indexes=self.operation_config.key_indexes,
+                keys_only=True,
+            )
+            logger.info("soap_api_diff complete", extra={"soap_api_diff": diff_results})
```

Comment:
I don't think I see it in what is returned, but could we make sure there is a field in the diff that just says whether it matches or not as a boolean for the whole model?

Just so we have something for a simple graph of percentage of records that match for a given endpoint that doesn't need to check things like whether the results themselves are empty or not.